x-pack/legacy/plugins/siem/public/components/header_page/editable_title.tsx

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import styled, { css } from 'styled-components';
+
+import {
+  EuiButton,
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFieldText,
+  EuiButtonIcon,
+} from '@elastic/eui';
+
+import { Title } from './title';
+
+const StyledEuiButtonIcon = styled(EuiButtonIcon)`
+  ${({ theme }) => css`
+    margin-left: ${theme.eui.euiSize};
+  `}
+`;
+
+StyledEuiButtonIcon.displayName = 'StyledEuiButtonIcon';
+
+interface Props {
+  submitTitle: string;
```

Comment:
do you think we will be reusing this component in different places? maybe we could just hardcode `submitTitle` and `cancelTitle` instead of using props?

Reply:
I do not know if the component will be used in different places. I abstract it with that in mind. Ok, let's hardcode the titles and if there is a need in the feature we will refactor.

Reply:
What about translations? Should be moved inside `header_page` ?